com.unity.render-pipelines.universal/Documentation~/universalrp-builtin-feature-comparison.md

```diff
@@ -104,7 +104,7 @@ This table provides an overview of the current features supported in the Univers
 | Order In Layer                                               | Not supported                                                | Yes                                                          |
 | ***Render Pipeline Hooks***                                  |                                                              |                                                              |
 | Camera.RenderWithShader                                      | Yes                                                          | Not supported                                                |
-| Camera.AddCommandBuffer*<br/>(Camera.Remove[All]CommandBuffer*) | Yes                                                          | Not supported                                                |
+| Camera.AddCommandBuffer*<br/>(Camera.Remove[All]CommandBuffer*) | Yes                                                       | Not supported                                                |
 | Camera.Render                                                | Yes                                                          | Not supported                                                |
 | Light.AddCommandBuffer*<br/>(LightRemove[All]CommandBuffer*) | Yes                                                          | Not supported                                                |
 | OnPreCull                                                    | Yes                                                          | Not supported                                                |
```

Comment:
OnPreCull and OnPreRender as replaced by RenderPipelineManager.BeginCameraRendering

Reply:
OnPostRender you can say it's replaced by ScriptableRenderPass as well.